constants/src/main/java/org/jboss/pnc/enums/ResultStatus.java

```diff
@@ -17,7 +17,13 @@
  */
 package org.jboss.pnc.enums;
 
+/**
+ * Status of generic result of some operation or task.
+ */
 public enum ResultStatus {
+    /**
+     * The operation was successful.
+     */
     SUCCESS(true), FAILED(false), SYSTEM_ERROR(false);
```

Comment:
It looks like the "The operation was successful" doc is for all three statuses in the line.

Reply:
What should be the descriptions for the two other states? Without description the formatter puts it on one line.
cc @michalovjan

Reply:
+1 these two should be documented as well

Reply:
The operation failed.

Reply:
@michalovjan for both it should read "The operation failed"? Why are there 2 then?